### PR TITLE
Add sqlite3 to msvc build

### PR DESCRIPTION
**Pull request by anosach-luxoft**
- Change cmake files to include sqlite3 headers
- Fix sqlite_wrapper build

Sqlite3 library should be installed and environment variable SQLITE_DIR point to
Sqlite3 directory

### CODE REVIEW

**Comment by anosach-luxoft** [comment]
@nk0leg, @AGaliuzov, @pestOO, please review

**Comment by AGaliuzov** [comment]
reviewed

File: src/components/utils/src/sqlite_wrapper/sql_query.cc [hunk 1]
```diff
@@ -53,7 +53,7 @@ bool SQLQuery::Prepare(const std::string& query) {
   Finalize();
   sync_primitives::AutoLock auto_lock(statement_lock_);
   if (statement_) return false;
-  error_ = sqlite3_prepare(db_.conn(), query.c_str(), query.length(),
+  error_ = sqlite3_prepare(db_.conn(), query.c_str(), (int)query.length(),
```
Comment (pestOO) [inline src/components/utils/src/sqlite_wrapper/sql_query.cc]: May use static_cast according to google style?

**Comment by pestOO** [comment]
Reviewed

**Pull request merged**